prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -79,15 +79,27 @@ public class PharmanetValidationRule : AutomaticAdjudicationRule
 
         public override async Task<bool> ProcessRule(Enrollee enrollee)
         {
-            if (enrollee.Certifications == null || !enrollee.Certifications.Any())
+            var certifications = enrollee.Certifications.Where(c => c.License.Validate);
+
+            // If enrollee choses device provider, add device provider as a licence and check it against PharmaNet
+            if (enrollee.HasCareSetting(CareSettingType.DeviceProvider) || enrollee.DeviceProviderIdentifier != null)
+            {
+                certifications = certifications.Append(new Certification
+                {
+                    License = new License { Prefix = "Prefix place holder" },
+                    LicenseNumber = enrollee.DeviceProviderIdentifier
+                });
+            }
+
+            if (!certifications.Any())
             {
-                // No certs to verify
+                // No certificationss to verify
                 return true;
             }
```

Comment:
This guard cause isn't really necessary any more; it's more or less just been hanging around since the first version of the Rule and we can remove it now.

Reply:
Removed